TestApp/appcenter-post-clone.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
We don't run AC Build on TestApp, only on DemoApp. No need to add this post clone script to TestApp.

Reply:
But we do that on other platforms, at least Android and Xamarin where we test building develop TestApp builds on every push.

Reply:
I guess it's not enough as we need to call `./update-npm-packages` for TestApp. Since it's out of scope I would remove this file for now unless you can make it work quickly on AppCenter build on your branch.

Reply:
@guperrot The reason we don't run AC Build is AC Build is on npm@3 which does not recognize our local dependency in the format of `"xxx": "file:xxx-1.6.0.tgz"` (see [link](https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/TestApp/package.json#L11)).

Reply:
@bmourat let's delete the file for now then, thanks.